Bottom: 4ea9c531bb71f49dfa3a2778fe4ab49d5dd3df1f
Top:    56713c45e7181516168044a4f54083f4f577dd04
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 19:14:01 +0000

Refresh of convert-layer-and-group

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index fb6c0a0..af87448 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -740,9 +740,6 @@ object3d_from_board_outline (void)
 {
   POLYAREA *board_outline = board_outline_poly (true);
   POLYAREA *pa;
-  appearance *board_appearance;
-  appearance *top_bot_appearance;
-  GList *objects;
 
 #if 0
   return object3d_from_soldermask_within_area (board_outline, TOP_SIDE);
@@ -761,6 +758,10 @@ object3d_from_board_outline (void)
     }
   while ((pa = pa->f) != board_outline);
 
+  appearance *board_appearance;
+  appearance *top_bot_appearance;
+  GList *objects;
+
   board_appearance = make_appearance ();
   top_bot_appearance = NULL;
   appearance_set_color (board_appearance,   1.0, 1.0, 0.6);
diff --git a/src/misc.c b/src/misc.c
index c7bdef2..0e46b98 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -614,7 +614,7 @@ SetTextBoundingBox (FontType *FontPtr, TextType *Text)
 {
   SymbolType *symbol = FontPtr->Symbol;
   unsigned char *s = (unsigned char *) Text->TextString;
-  unsigned int i;
+  int i;
   int space;
 
   Coord minx, miny, maxx, maxy, tx;
